reference/configuration/doctrine.rst

```diff
@@ -50,7 +50,8 @@ The following block shows all possible configuration keys:
                 user:                 user
                 password:             secret
                 driver:               pdo_mysql
-                # if the url option is specified, it will override the above config
+                # until doctrine-bundle 2.2, the url option overrides the above config
+                # since doctrine-bundle 2.3 it is recommended to set override_url: true, which allows the above config to override the parts of the url
```

Comment:
Actually not, as we're going to deprecate override_url

Reply:
are you reverting the feature? or will you do doctrine-bundle 3 and force the behaviour?

any suggestion what to say here?

Reply:
We are going to revert the feature yes. Maybe we should just keep the existing sentence

Reply:
ah, then should i change the note to say that DoctrineBundle 2.3 introduced override_url but the feature will be retired and recommend to set override_url to false? 

also, if you will revert anyways, why not change the recipe to not set override_url (or set it to false)?

Reply:
we merged the change yesterday :) https://github.com/symfony/recipes/pull/939
maybe there is nothing to do...